### PR TITLE
Edit styleguide tables

### PR DESCRIPTION
**Pull request by Sjones352**
connects to #2144

### CODE REVIEW

File: client/app/components/Table.jsx [hunk 1]
```diff
@@ -127,7 +127,7 @@ export default class Table extends React.PureComponent {
       bodyClassName = '',
       rowClassNames = this.defaultRowClassNames,
       getKeyForRow,
-      slowReRendersAreOk,
+      slowReRendersAreOk = true,
```
Comment (mmlumba) [inline client/app/components/Table.jsx]: The boolean goes in the `<Table>` component, not the prop declaration, ie:
`<Table
name="My Table"
slowReRendersAreOk=true
/>`

Comment (Sjones352) [inline client/app/components/Table.jsx]: So i put the boolean in both tables and I am still getting the same error`<Table columns={columnsWithAction} rowObjects={rowObjects} summary={summary} slowReRendersAreOk={true} />`

Comment (NickHeiner) [inline client/app/components/Table.jsx]: Is there another `Table` being used?

Comment (Sjones352) [inline client/app/components/Table.jsx]: @NickHeiner yes there was another table in Typography. Fixed it so no more errors

File: client/app/containers/StyleGuide/StyleGuideTables.jsx [hunk 2]
```diff
@@ -9,13 +9,13 @@ export default function StyleGuideTables() {
   // List of objects which will be used to create each row
   let rowObjects = [
     { name: 'Shane',
-      spiritAnimal: 'Hamster',
+      patronus: 'Hamster',
```
Comment (mmlumba) [inline client/app/containers/StyleGuide/StyleGuideTables.jsx]: sorry, can you make the property names consistent with the row?

**Pull request merged**